src/ex1/index.html

```diff
@@ -1,14 +1,60 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-    <title>Exercise 1</title>
-    <link rel="stylesheet" href="style.css" />
+  <title>TODO RAVE</title>
+  <link rel="stylesheet" href="style.css" />
 </head>
-<body>
 
-<h1>Todo list</h1>
+<body>
+  <div id="myDIV" class="header">
```

Comment:
important: the convention for css class names and ids are kebab-case and not camel case

```suggestion
  <div id="my-div" class="header">
```